src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocCommentsTokenTypes.java

```diff
@@ -881,7 +881,36 @@ public final class JavadocCommentsTokenTypes {
     public static final int EXTENDS = JavadocCommentsLexer.EXTENDS;
 
     /**
-     * Keyword {@code super} in type parameters.
+     * {@code SUPER} represents the {@code super} keyword inside a generic
+     * wildcard bound (e.g., {@code ? super Number}).
+     *
+     * <p><b>Example:</b> {@code {@link java.util.List<? super Integer> list}
+     * of any supertype of Integer}</p>
+     *
+     * <p><b>Tree:</b></p>
+     * <pre>{@code
+     * JAVADOC_INLINE_TAG -> JAVADOC_INLINE_TAG
+     * `--LINK_INLINE_TAG -> LINK_INLINE_TAG
+     *     |--JAVADOC_INLINE_TAG_START -> &#123;@
```

Comment:
can you put a space in between on `{` and `@`
I see other use this hack to bypass poblems

Reply:
see a hack here https://github.com/checkstyle/checkstyle/pull/18284/changes#diff-3ea25795275336f80846bbb84b06f0da3fbc43656a8b77b7f0c821d7e60aa1adR722